GOVERNANCE.rst

```diff
@@ -0,0 +1,139 @@
+TC members merge/close PRs and issues, and moderate (including block/mute) bad actors.
+The TC can propose governance changes to the SC.
+
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Discussion in the TC usually happens in the developer meetings. Developer meetings can be scheduled by any member of the SC or TC.
+
+If someone calls for a vote to make a decision: majority based on the cast votes; we need 50% of the committee participating to vote. In the absence of a quorum, the SC will decide according to its voting rules.
+
+Votes are cast in a non-confidential manner.
+Decisions are documented in the developer meeting notes and/or on the GitHub repository.
+
+TC members can individually appoint new contributors, unless a vote is called on an individual.
+
+Appointment Process
+^^^^^^^^^^^^^^^^^^^
+
+TC members are the maintainers of AMReX.
+As a TC member, regularly attending and contributing to the developer meetings is expected.
+
+One is appointed to the TC by the steering committee, in a unanimous vote, or by majority vote of the TC. The SC can veto appointments.
+Steering committee members can also be TC members.
+
+TC members can resign or be removed by majority vote by either TC or SC, e.g., due to inactivity, bad acting or other reasons.
+
+
+Contributors
+------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+See: `GitHub contributors <https://github.com/AMReX-Codes/amrex/graphs/contributors>`__
+
+Role
+^^^^
+
+Contributors are valuable, vetted developers of AMReX.
+Contributions can be in many forms and not all need to be code contributions.
+Examples include code pull requests, support in issues & user discussions, writing and updating documentation, writing tutorials, visualizations, R&D on algorithms, testing and benchmarking, etc.
+Contributors can participate in developer meetings and weigh in on discussions.
+Contributors can "triage" (add labels) to pull requests, issues, and GitHub discussion pages.
+Contributors can comment and review PRs (but not merge).
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Contributors can individually decide on classification (triage) of pull requests, issues, and GitHub discussion pages.
+
+Appointment Process
+^^^^^^^^^^^^^^^^^^^
+
+Appointed after contributing to AMReX (see above) by any member of the TC.
+
+The role can be lost by resigning or by decision of an individual TC or SC member, e.g., due to inactivity, bad acting or other.
+
+
+Former Members
+--------------
+
+"Former members" are *not* tracked as a governance role in AMReX.
+Instead, former (e.g., inactive) contributors are acknowledged separately in GitHub contributor tracking, the AMReX documentation, references, etc. as appropriate.
+
+Former members of SC, TC and Contributors are not kept in the roster, since committee role rosters should reflect currently active members and the responsible governance body.
```

Comment:
Former members of the SC or TC also do not play any role in the governance of AMReX.